authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -217,26 +218,75 @@ func getResultCacheKey(subRevReq *authzv1.SubjectAccessReviewSpec) string {
 	return cacheKey
 }
 
-func prepareCheckAccessRequestBody(req *authzv1.SubjectAccessReviewSpec, clusterType, resourceId string) (*CheckAccessRequest, error) {
-	checkaccessreq := CheckAccessRequest{}
+func prepareCheckAccessRequestBody(req *authzv1.SubjectAccessReviewSpec, clusterType, resourceId string, retrieveGroupMemberships bool) (*CheckAccessRequest, error) {
+	/* This is how sample SubjectAccessReview request will look like
```

Comment:
thank you so much for adding a sample. it'd be great if you can format the json a bit :)